README.md

```diff
@@ -189,21 +189,21 @@ This is a **current** feature status list:
-| Trace SDK                   | v<!--VERSION_STABLE-->1.13.0<!--/VERSION_STABLE-->           |
-| Context                     | v<!--VERSION_STABLE-->1.13.0<!--/VERSION_STABLE-->           |
-| Baggage                     | v<!--VERSION_STABLE-->1.13.0<!--/VERSION_STABLE-->           |
-| Jaeger Trace Exporter       | v<!--VERSION_STABLE-->1.13.0<!--/VERSION_STABLE-->           |
-| Zipkin Trace Exporter       | v<!--VERSION_STABLE-->1.13.0<!--/VERSION_STABLE-->           |
-| OTLP Exporter (Spans)       | v<!--VERSION_STABLE-->1.13.0<!--/VERSION_STABLE-->           |
-| Metrics API                 | v<!--VERSION_STABLE-->1.13.0<!--/VERSION_STABLE-->           |
-| OTLP Exporter (Metrics)     | v<!--VERSION_UNSTABLE-->1.13.0-alpha<!--/VERSION_UNSTABLE--> |
-| Metrics SDK                 | v<!--VERSION_UNSTABLE-->1.13.0-alpha<!--/VERSION_UNSTABLE--> |
-| OTLP Exporter (Logs)        | v<!--VERSION_UNSTABLE-->1.13.0-alpha<!--/VERSION_UNSTABLE--> |
-| Logs SDK                    | v<!--VERSION_UNSTABLE-->1.13.0-alpha<!--/VERSION_UNSTABLE--> |
-| Prometheus Metrics Exporter | v<!--VERSION_UNSTABLE-->1.13.0-alpha<!--/VERSION_UNSTABLE--> |
-| OpenTracing Bridge          | v<!--VERSION_UNSTABLE-->1.13.0-alpha<!--/VERSION_UNSTABLE--> |
-| OpenCensus Bridge           | v<!--VERSION_UNSTABLE-->1.13.0-alpha<!--/VERSION_UNSTABLE--> |
+| Trace API                   | v<!--VERSION_STABLE-->1.15.0-SNAPSHOT<!--/VERSION_STABLE-->           |
+| Trace SDK                   | v<!--VERSION_STABLE-->1.15.0-SNAPSHOT<!--/VERSION_STABLE-->           |
+| Context                     | v<!--VERSION_STABLE-->1.15.0-SNAPSHOT<!--/VERSION_STABLE-->           |
+| Baggage                     | v<!--VERSION_STABLE-->1.15.0-SNAPSHOT<!--/VERSION_STABLE-->           |
+| Jaeger Trace Exporter       | v<!--VERSION_STABLE-->1.15.0-SNAPSHOT<!--/VERSION_STABLE-->           |
+| Zipkin Trace Exporter       | v<!--VERSION_STABLE-->1.15.0-SNAPSHOT<!--/VERSION_STABLE-->           |
+| OTLP Exporter (Spans)       | v<!--VERSION_STABLE-->1.15.0-SNAPSHOT<!--/VERSION_STABLE-->           |
+| Metrics API                 | v<!--VERSION_STABLE-->1.15.0-SNAPSHOT<!--/VERSION_STABLE-->           |
+| OTLP Exporter (Metrics)     | v<!--VERSION_UNSTABLE-->1.15.0-SNAPSHOT-alpha<!--/VERSION_UNSTABLE--> |
+| Metrics SDK                 | v<!--VERSION_UNSTABLE-->1.15.0-SNAPSHOT-alpha<!--/VERSION_UNSTABLE--> |
+| OTLP Exporter (Logs)        | v<!--VERSION_UNSTABLE-->1.15.0-SNAPSHOT-alpha<!--/VERSION_UNSTABLE--> |
+| Logs SDK                    | v<!--VERSION_UNSTABLE-->1.15.0-SNAPSHOT-alpha<!--/VERSION_UNSTABLE--> |
+| Prometheus Metrics Exporter | v<!--VERSION_UNSTABLE-->1.15.0-SNAPSHOT-alpha<!--/VERSION_UNSTABLE--> |
+| OpenTracing Bridge          | v<!--VERSION_UNSTABLE-->1.15.0-SNAPSHOT-alpha<!--/VERSION_UNSTABLE--> |
+| OpenCensus Bridge           | v<!--VERSION_UNSTABLE-->1.15.0-SNAPSHOT-alpha<!--/VERSION_UNSTABLE--> |
```

Comment:
yikes, idk what's going on with this and all of the `apidiffs` updates. This is what the `./gradlew check` did...

Reply:
Sorry about that @jonahaapala, you got caught in a race condition with our release. I pushed a commit to this branch to pull in HEAD to fix it